hatchery/gen3licenseusermaps.go

```diff
@@ -382,6 +383,43 @@ var getKubeClientSet = func() (clientset kubernetes.Interface, err error) {
 
 }
 
+// TODO: descide if this stays here or gets moved to gen3licenseusermaps
+var getLicenseString = func(Config *FullHatcheryConfig, hash string) (string, error) {
+	// get the file_path and fileId from config
+	file_path := Config.ContainersMap[hash].License.FilePath
+	filePathConfigs, err := getLicenceFilePathConfigs()
```

Comment:
```suggestion
	filePathConfigs, err := getLicenseFilePathConfigs()
```
FYI there is a typo in the original function name and we should fix that function as well

Reply:
Good catch, thank you, this has been corrected.